prov/verbs/src/ep_rdm/verbs_utils.h

```diff
@@ -64,14 +64,12 @@ struct fi_ibv_msg_ep;
 
 #define FI_IBV_RDM_CM_THREAD_TIMEOUT (100)
 #define FI_IBV_RDM_MEM_ALIGNMENT (64)
-#define FI_IBV_RDM_BUF_ALIGNMENT (4096)
+#define FI_IBV_RDM_BUF_ALIGNMENT (4096) /* TODO: Page size */
 
 #define FI_IBV_RDM_TAGGED_DFLT_BUFFER_NUM (8)
 
 #define FI_IBV_RDM_DFLT_BUFFER_SIZE					\
-	((8 * 1024 + FI_IBV_RDM_BUFF_SERVICE_DATA_SIZE) +		\
-	 (8 * 1024 + FI_IBV_RDM_BUFF_SERVICE_DATA_SIZE) %		\
-	  FI_IBV_RDM_BUF_ALIGNMENT)
+	(3 * FI_IBV_RDM_BUF_ALIGNMENT)
 
 #define FI_IBV_RDM_DFLT_BUFFERED_SSIZE					\
 	(FI_IBV_RDM_DFLT_BUFFER_SIZE -					\
```

Comment:
These lines were changed anyway because of type changing or something else. That is more preferable alignment of args, to opening bracket by spaces or common tabulation?